### ISSUE TITLE
Doesn't work with Composer >=2.5 (?)

### ISSUE THREAD

**Issue by chekalsky** [avto-dev/composer-cleanup-plugin#12]
Hey, we've started to get errors during composer install:

```
Error: AvtoDev\Composer\Cleanup\Plugin::makeClean(): Argument #1 ($package_path) must be of type string, null given, called in /home/runner/work/***/***/vendor/avto-dev/composer-cleanup-plugin/src/Plugin.php on line 147

In Plugin.php line 167:
                                                                               
  [TypeError]                                                                  
  AvtoDev\Composer\Cleanup\Plugin::makeClean(): Argument #1 ($package_path) m  
  ust be of type string, null given, called in /home/runner/work/***/***/ve  
  ndor/avto-dev/composer-cleanup-plugin/src/Plugin.php on line 147             
```

Rolling back composer to version 2.2.20 seems to help, so the issue is reproducable with Composer >=[2.5.0](https://github.com/composer/composer/releases/tag/2.5.0)

**Comment by eldario** [comment]
## Fixed in #13 

You can check the new version and close the issue if the error is no longer observed

**Comment by chekalsky** [comment]
@eldario thank you, the issue is no longer observed
